easybuild/tools/build_log.py

```diff
@@ -107,8 +107,9 @@ def nosupport(self, msg, ver):
 
     def error(self, msg, *args, **kwargs):
         """Print error message and raise an EasyBuildError."""
-        newMsg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
-        fancylogger.FancyLogger.error(self, newMsg, *args, **kwargs)
+        newMsg = ("EasyBuild crashed with an error %s: %s"
+                  % (self.caller_info(), (msg % args)))
```

Comment:
Yes, but since args have already been interpolated into `newMsg`, there's no need to pass them to the logger: `newMsg` no longer contains any `%s` format specifiers.